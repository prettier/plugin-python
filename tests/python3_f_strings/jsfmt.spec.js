run_spec(__dirname, ["python"], ">=3.6");
