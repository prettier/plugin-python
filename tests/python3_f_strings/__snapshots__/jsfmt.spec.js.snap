// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`f_strings.py 1`] = `
width = 10
precision = 4
value = decimal.Decimal("12.34567")
f"result: {value:{width}.{precision}}"
rf"result: {value:{width}.{precision}}"
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
width = 10

precision = 4

value = decimal.Decimal("12.34567")

f"result: {value:{width}.{precision}}"

rf"result: {value:{width}.{precision}}"

`;
